controls/SV-221580.rb

```diff
@@ -44,4 +44,15 @@
   tag legacy: ["SV-57625","V-44791"]
   tag cci: ["CCI-001166"]
   tag nist: ["SC-18 (1)"]
-end
\ No newline at end of file
+  
+  describe.one do
+    describe registry_key('HKEY_LOCAL_MACHINE\Software\Policies\Google\Chrome\') do
```

Comment:
The last backslash in the registry_key path, the backslash after Chrome, serves as an escape character and have escaped the single quote after it. Consider removing the backslash or adding an extra backlash to fix the syntax.